dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/AI.cs

```diff
@@ -22,21 +22,20 @@ namespace Microsoft.TeamsAI.AI
     public class AI<TState> where TState : ITurnState<StateBase, StateBase, TempState>
     {
         private readonly IActionCollection<TState> _actions;
-        private readonly AIOptions<TState> _options;
 
         public AI(AIOptions<TState> options, ILogger? logger = null)
         {
             Verify.ParamNotNull(options);
 
-            _options = options;
+            Options = options;
```

Comment:
What's the reason for this rename?

Reply:
To fix the warning [IDE0032](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0032).